client/src/components/App/App.css

```diff
@@ -0,0 +1,89 @@
+.product-overview {
```

Comment:
@qzyan Are we not using the normalize.css anymore for the App?

Reply:
This is the CSS code Allen shared in the example CSS file. Use it as a skeleton of the whole app.